tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -185,7 +185,7 @@ static void SavePixelsToFile(LPCVOID pPixels, DXGI_FORMAT format, UINT32 m_width
   VERIFY_SUCCEEDED(ctx.Init());
   VERIFY_SUCCEEDED(CoCreateInstance(CLSID_WICImagingFactory, NULL, CLSCTX_INPROC_SERVER, IID_IWICImagingFactory, (LPVOID*)&pFactory));
   VERIFY_ARE_NOT_EQUAL(pFormat, Vals + _countof(Vals));
-  VERIFY_SUCCEEDED(pFactory->CreateBitmapFromMemory(m_width, m_height, pFormat->PixelFormat, m_width * pFormat->PixelSize, m_width * m_height * pFormat->PixelSize, (BYTE *)pPixels, &pBitmap));
+  VERIFY_SUCCEEDED(pFactory->CreateBitmapFromMemory(m_width, m_height, pFormat->PixelFormat, m_width * pFormat->PixelSize, m_width * m_height * pFormat->PixelSize, const_cast<BYTE *>((const BYTE*)(pPixels)), &pBitmap));
```

Comment:
I see now. This is an interface we don't control and it requires deconstifying. I'm not inclined to make that pointer non-const to satisfy this. So I think this is one of the rare appropriate deconst casts.